src/components/DevNavMenu.tsx

```diff
@@ -279,6 +279,10 @@ const DevNavMenu = {
           {
             title: 'Genesis Configuration',
             link: '/how-to-guides/v3/basics/genesis/',
+          },
+		  {
+            title: 'Custom Chain Specifications',
+            link: '/how-to-guides/v3/basics/genesis/',
```

Comment:
@gilescope this is what is needed to expose things in the nav (this file in general)